library/src/containers/Messages/NewMessages.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { MessageComponent } from './NewMessage';
+import { Toggle } from '../../components';
+
+import { bemClasses } from '../../helpers';
+import { useSpec } from '../../store';
+import { CONTAINER_LABELS, MESSAGES_TEXT } from '../../constants';
+
+export const MessagesComponent: React.FunctionComponent = () => {
+  const messages = useSpec().allMessages();
+
+  if (!messages.size) {
+    return null;
+  }
+
+  const className = CONTAINER_LABELS.SCHEMAS;
+  const header = <h2>{MESSAGES_TEXT}</h2>;
+
+  const messagesList = (
+    <ul>
+      {Array.from(messages).map(([key, msg], idx) => (
+        <li key={key}>
+          <MessageComponent message={msg} index={idx + 1} key={idx} />
+        </li>
+      ))}
+    </ul>
+  );
+
+  return (
+    <section
+      className={bemClasses.element(className)}
+      id={bemClasses.identifier([className])}
```

Comment:
Same thing with BEM here.

Reply:
I know, but section only wraps content like this:

![image](https://user-images.githubusercontent.com/20404945/112468328-68d77d00-8d68-11eb-9674-bae1fe9d3975.png)

I wanna make styles here asyncapi/shape-up-process#86